components/ui/ProgressFreemiumComponent.tsx

```diff
@@ -0,0 +1,34 @@
+import { Indicator, Root } from "@radix-ui/react-progress";
```

Comment:
I could see us using this type of component across the Skillify site - thinking badge completion progress or even for course completion

What if we generalize this so it's not just for Freemium? Perhaps we could call it ProgressComponent and it could take in two props like currentValue and totalValue or something general like that

Reply:
I just changed it.  great idea.  the initial reason I didn't do this was because we had a ProgressBar.tsx, which is really the same thing.  I found it after I created this one, when we moved it into the UI folder.  I didn't want it to become confusing.  So I think the names you proposed are much better -- I just added a comment to distinguish between them.  let me know if you think that's a good idea.